.travis.yml

```diff
@@ -4,24 +4,42 @@ scala:
 - 2.11.7
 services:
   - docker
+cache:
+  directories:
+  - node_modules
 before_install:
+- nvm install 6.0.0
+- npm install -g yarn
 - docker pull java:8u92-jdk-alpine
 - wget https://repo.typesafe.com/typesafe/ivy-releases/org.scala-sbt/sbt-launch/0.13.11/sbt-launch.jar
 - echo 'java -Xms512M -Xmx1536M -Xss1M -XX:+CMSClassUnloadingEnabled -XX:MaxPermSize=256M -jar sbt-launch.jar "$@"' > sbt.sh
+- echo 'ctp.projectKey=${SUNRISE_IT_CTP_PROJECT_KEY}' > conf/dev.conf
+- echo 'ctp.clientId=${SUNRISE_IT_CTP_CLIENT_ID}' >> conf/dev.conf
+- echo 'ctp.clientSecret=${SUNRISE_IT_CTP_CLIENT_SECRET}' >> conf/dev.conf
+- "docker run --name=sunrise-test -it -d -p 9000:9000 -e SUNRISE_IT_CTP_PROJECT_KEY=$SUNRISE_IT_CTP_PROJECT_KEY -e SUNRISE_IT_CTP_CLIENT_ID=$SUNRISE_IT_CTP_CLIENT_ID -e SUNRISE_IT_CTP_CLIENT_SECRET=$SUNRISE_IT_CTP_CLIENT_SECRET --volume=$(pwd):/src --volume=/home/travis/.ivy2:/ivy2  --volume=/home/travis/.sbt:/sbt -w /src java:8u92-jdk-alpine sh"
+- "docker exec sunrise-test sh sbt.sh compile"
+- "docker exec -itd sunrise-test sh sbt.sh run"
```

Comment:
I think `testProd` instead of `run` makes sense here so we really have the production settings (huge impact on DI and performance). 

https://www.playframework.com/documentation/2.5.x/Deploying#Running-a-test-instance